### DISCUSSION TITLE
On error occured, why not wait for unfinished targets to finish.

### DISCUSSION THREAD

**Discussion by xinstein** [ropensci/targets#533] [Help]
I am running into a problem which prevents me from using targets effectively:
I have a big graph and I run it in parallel using tar_make_future,
when one of the targets encountered error, there's a good chance that there are multiple parallel targets which are nearly finished, I think it would be more economical if we let them finish than make them stop immediately and start over on next make.

Currently there are three options to error handling: "stop"/"workspace"/"continue":
"stop"/"workspace" : These options will cause the concurrent jobs to stop immediately and completely start over on next launch, just as I described;
"continue": This will simply skip any failed targets and launch downstream jobs. I can't think of any good use case for this option.

**Comment by wlandau** [comment]
Converting to an issue: #534.
